### ISSUE TITLE
Can't close contextual menu via KB (expected 'esc' to close the menu)

### ISSUE THREAD

**Issue by AviadGigi** [OfficeDev/office-ui-fabric-react#11077]
### Environment Information

- **Package version(s)**: latest
- **Browser and OS versions**: chrome win10
> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Can't close contextual menu with keyboard

#### Actual behavior:

go to fabric site https://developer.microsoft.com/en-us/fabric#/controls/web/contextualmenu 
using only the keyboard open a contextual menu, and try close it without selecting any option 

#### Expected behavior:

being able to close the menu using 'esc' same as popup component behavior

**Comment by micahgodbolt** [comment]
problem is that contextual menu is passing down an event during onDismiss that already has 'preventDefault' set to true. I have a PR set up to remove that, while leaving stopPropegation, but need to figure out what effect that might have.

**Comment by micahgodbolt** [comment]
Fixed by #11077
